app/components/user_item/user_item.tsx

```diff
@@ -80,7 +80,6 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
             color: changeOpacity(theme.centerChannelColor, 0.64),
             fontSize: 15,
             fontFamily: 'OpenSans',
-            flexShrink: 5,
```

Comment:
What is the impact of this?

Reply:
Have you tested this changes in other places where this component is used?

Reply:
The idea is to shrink this element based on the current elements count, so the space is better distributed. Tested id a well in the autocomplete.